config/routes.rb

```diff
@@ -1,2 +1,6 @@
 Rails.application.routes.draw do
+  scope module :api, defaults { format: 'json'} do
+    namespace :v1 do # V1 resources will go here
```

Comment:
One item Edward and I forgot to mention was our Style Guide, which is good reference --and what we've used when definiing the Rubocop rules.

One of the recommendations is https://github.com/IntrepidPursuits/web-style-guides/blob/master/style.md#formatting

to avoid inline comments.

We're not big on comment, in general unless something very out of teh ordinary is done.